docs/python.rst

```diff
@@ -74,3 +74,52 @@ like this::
     - python3-requests.json
     # (other modules go here)
 
+Installing dependencies without flatpak-pip-generator using setuptools or pipenv
+--------------------------------------------------------------------------------
+
+As an alternative to flatpak-pip-generator a slightly more pythonic approach would
+be using setup.py or requirements.txt directly.
+To do this with ``setuptools`` you can do so as follows:
+
+.. code-block:: yaml
+
+  app-id: org.flatpak.PackagingDemo
+  runtime: org.freedesktop.Platform
+  runtime-version: '1.6'
+  sdk: org.freedesktop.Sdk
+  command: app.py
+  finish-args:
+  - "--share=network"
```

Comment:
You need to document that this means it cannot run anywhere `flatpak-builder` is sandboxed, like Flathub's builders.

Reply:
Could you go into a little more detail here. I'm copying the scripts into the package here using setup.py to install it inside. Unless there is some other step needed

Reply:
Sorry I commented on the wrong line. Below `build-args` is forbidden in a sandbox build, which will always block network access.